website/signup/earthly-ci/index.html

```diff
@@ -14,96 +14,115 @@
-                <div>
-                    <div class="text-3xl lg:text-4xl font-semibold">
-                        Try Earthly CI
-                    </div>
-                    <div class="text-xl lg:text-2xl text-slate-500 pt-1 pb-3">
-                        Sign up here, and we'll contact you shortly to begin onboarding.
-                    </div>
-                    <div class="formkit-content">
-                        <div class="formkit-text " style="color:#060606" type="Text" id="425443429"
-                            content="&lt;p&gt;Sign Up for Earthly CI&lt;/p&gt;"
-                            element_id="b9d0d453-1beb-4c25-9a90-fc92314a526b" order="0" region="content">
+        <div class="pt-28">
+          <div class="text-3xl lg:text-4xl font-semibold pb-3">
+            Try Earthly CI
+          </div>
+          <div class="lg:w-1/2 border bg-[#eafaf4] px-4 lg:px-10 py-2 lg:py-6 rounded-lg">
+            <div class="lg:block text-gray-600">
+              <div class="pb-4 text-gray-600">
+                  Earthly CI is a super simple CI/CD platform that gives you fast, repeatable
+                  pipelines that run exactly the same in CI as on your laptop.
+              </div>
+              <div class="flex pb-2">
+                  <img src="../../assets/svg/done-check.svg" alt="Checked bullet" class="" />
+                  <div class="px-2">Repeatable CI pipelines</div>
+              </div>
+              <div class="flex pb-2">
+                  <img src="../../assets/svg/done-check.svg" alt="Checked bullet" class="" />
+                  <div class="px-2">Ridiculously fast builds</div>
+              </div>
+              <div class="flex pb-2">
+                  <img src="../../assets/svg/done-check.svg" alt="Checked bullet" class="" />
+                  <div class="px-2">Super simple</div>
+              </div>
+              <div class="flex pb-2">
+                  <img src="../../assets/svg/done-check.svg" alt="Checked bullet" class="" />
+                  <div class="px-2">Great for monorepos and polyrepos</div>
+              </div>
+              <div class="flex pb-2">
+                  <img src="../../assets/svg/done-check.svg" alt="Checked bullet" class="" />
+                  <div class="px-2">Remote build runners built-in</div>
+              </div>
+            </div>
+          </div>
+          <div class="py-2 lg:py-6">
+            <div class="text-xl lg:text-2xl text-slate-500 pb-1">
+              <span class="font-bold">If you have less than 5 users</span>, click here to go to our guide to self-serve Earthly CI.
+            </div>
+            <div class="text-base lg:text-lg pb-3">
+              This will take you to our documentation and give you steps to sign up for and onboard onto Earthly CI.
+            </div>
+            <form id="earthly-ci-self-serve-form" method="GET" action="https://docs.earthly.dev/earthly-cloud/earthly-ci#getting-started">
```

Comment:
I would link to the beginning of the page here, to also capture the introduction.

```suggestion
            <form id="earthly-ci-self-serve-form" method="GET" action="https://docs.earthly.dev/earthly-cloud/earthly-ci">
```